code/controllers/configuration/entries/jobs.dm

```diff
@@ -0,0 +1,184 @@
+// This file pretty much just handles all of the interactions between jobconfig.toml and the codebase. This is started by work originating in SSconfig, so I'm okay with it being here.
+
+/// Initializes all of the config singletons for each job config type and adds it to the `job_config_datum_singletons` var list.
+/datum/controller/subsystem/job/proc/generate_config_singletons()
+	for(var/datum/job_config_type/config_datum as anything in subtypesof(/datum/job_config_type))
+		var/config_datum_key = initial(config_datum.name)
+		job_config_datum_configurables += config_datum_key
+		var/new_config_datum = new config_datum
+		job_config_datum_singletons[config_datum_key] = new_config_datum
+
+/// Called in jobs subsystem initialize if LOAD_JOBS_FROM_TXT config flag is set: reads jobconfig.toml (or if in legacy mode, jobs.txt) to set all of the datum's values to what the server operator wants.
+/datum/controller/subsystem/job/proc/load_jobs_from_config()
```

Comment:
Describing how something is called is often less interesting then what it does (and is further prone to drift). why do it here?